### DISCUSSION TITLE
User login gets disabled anytime I manually do a sync

### DISCUSSION THREAD

**Discussion by twinsmom71** [snipe/snipe-it#15905] [Q&A]
Wondering if anyone else is seeing this, if I do a manual LDAP sync all of my users who I had marked to have the ability to login get reset.  Any ideas?

**Comment by snipe** [comment]
Hi @twinsmom71 - sounds to me like you don't have the correct mapping set up in your LDAP settings. See the screenshot below:

<img width="769" alt="Screenshot 2024-12-02 at 4 15 29 PM" src="https://github.com/user-attachments/assets/ea4bd3f3-e8fc-4e35-baee-929e134c63bf">

**Reply by twinsmom71** [reply]
And what should it be set to if I only want to allow certain users to login and not everyone?

**Reply by snipe** [reply]
We don't currently support that, unfortunately.

**Reply by swift2512** [reply]
> And what should it be set to if I only want to allow certain users to login and not everyone?
I edit users via 'Bulk edit' feature and it looks like that ability for user to login doesn't change on LDAP sync. I've tested this today and all employees with disabled logins in AD are marked as able to login in Snipe-IT.
(Field 'LDAP Active Flag' is left blank in my install.)

**Reply by swift2512** [reply]
@twinsmom71 Just tested, for me it works both way. I've disabled login for user (via 'Bulk edit') who is Enabled in LDAP and ran LDAP Sync on SnipeIT- login didn't change to to Enabled.
You can always create a user group in admin settings with every permission denied - this way users can login, but can only see items assigned to them and nothing else.
